changelog/allow_casting_from_typetuple_to_typetuple.dd

```diff
@@ -0,0 +1,15 @@
+Allow casting from tuple to tuple
```

Comment:
Still not the clearest of descriptions. Wording it to say casting between compatible tuples would make more sense.